Vagrantfile

```diff
@@ -0,0 +1,25 @@
+require 'yaml'
+
+configuration = YAML.load(File.open(File.join(File.dirname(__FILE__), 'config.yaml'), File::RDONLY).read)
+
+Vagrant.configure(2) do |config|
+  source_path = configuration['sourcePath']
+  memory = configuration['memory']
+  web_port = configuration['webPort']
+  db_port = configuration['dbPort']
+
+  config.vm.box = 'ubuntu/trusty64'
```

Comment:
because we don't officially support jessie yet and wheezy is still being maintained by the Debian folks. Let's update this when we officially add support for jessie. nevertheless, OSIAM will definitely run on jessie and any other major linux distribution - the installation just might be slightly different.